dist/toggle-button/toggle-button.css

```diff
@@ -0,0 +1,104 @@
+:root {
+  --toggle-button-height-min: 72px;
+  --toggle-button-width-max: 342px;
+  --toggle-button-width-min: 224px;
+  --toggle-button-minimal-width-min: 72px;
+  --toggle-button-minimal-width-max: 168px;
+  --toggle-button-gallery-width-min: 168px;
+  --toggle-button-gallery-width-max: 342px;
+}
+.toggle-button {
+  background-color: var(--color-background-primary);
+  border: 1px solid var(--color-stroke-default);
+  border-radius: 8px;
+  box-shadow: 0 0 0 1px transparent;
+  color: var(--color-foreground-primary);
+  display: inline-flex;
+  font-size: 0.875rem;
+  margin-right: 8px;
+  margin-top: auto;
+  max-width: var(--toggle-button-width-max);
+  min-height: var(--toggle-button-height-min);
+  min-width: var(--toggle-button-width-min);
+  padding: 0 16px;
+  vertical-align: top;
+}
+.toggle-button:focus-visible {
+  outline-offset: 1px;
+  outline-style: outset;
+  outline-width: 2px;
+}
+.toggle-button[aria-pressed="true"] {
+  background-color: var(--color-background-secondary);
+  border: 1px solid var(--color-stroke-strong);
+  box-shadow: 0 0 0 1px var(--color-stroke-strong);
+  font-weight: bold;
+}
+.toggle-button__content {
+  align-self: center;
+  display: flex;
+  flex-direction: column;
+  text-align: left;
+}
+.toggle-button__icon {
+  margin-bottom: 16px;
+  margin-right: 16px;
+  margin-top: 16px;
+}
+.toggle-button__image {
+  align-self: center;
+  margin-bottom: 16px;
+  margin-right: 16px;
+  margin-top: 16px;
+}
+.toggle-button__image svg.icon {
```

Comment:
Should this be restricting the size of `img` tags instead? People who are using icons should be using `.toggle-button__icon`, not `.toggle-button__image`.

Reply:
This actually ensures that the image is vertically centered in the list view layout theme. But you did catch a scenario where a dev might use an actual <img>, which I failed to account for. Another great catch! I'll make an adjustment.